testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxContext.java

```diff
@@ -618,26 +367,48 @@ public T receivesEventsProducedBy(Object producerId, EventMessage first, EventMe
      *         a list of domain event to be dispatched to the Bounded Context
      * @return current instance
      */
-    private T receivesEvents(Collection<EventMessage> domainEvents) {
+    private BlackBoxContext receivesEvents(Collection<EventMessage> domainEvents) {
```

Comment:
Maybe it makes sense to include all these methods accepting a `Collection` of events and commands into public API of `BlackBoxContext`?

Reply:
Let's wait till there is a real need. We can easily add it later should we see such a need. For now we have a method which access only one `Event` or message. The API of `BlackBoxContext` allows chaining, so if we need a few of `Event`s, they can be passed sequentially. Also, a test suite can always perform a streaming API call (on the collection of events or commands) referencing corresponding method of `BlackBoxContext`.